inspire_disambiguation/core/ml/models.py

```diff
@@ -676,7 +676,7 @@ def score(self, test_uuids, labels):
             labels - list of labels (author id) for test dataset
         """
         all_uuids = np.vectorize(lambda x: x.signature_uuid)(self.X).flatten()
-        test_uuids_array = np.array(test_uuids)
+        test_uuids_array = np.array(list(test_uuids))
```

Comment:
Is this really needed?

Reply:
Yes, because it's set (I updated the docs)